client/actions/algorithms.js

```diff
@@ -22,6 +22,27 @@ export function algorithmsFetchDataSuccess(algorithms) {
   };
 }
 
+export function templatesHasErrored(hasErrored) {
```

Comment:
The template should be in a different file, not in algorithms.js. It would be clearer and easier to find if we have to use it again.